ui/components/multichain/pages/send/send.js

```diff
@@ -158,17 +199,29 @@ export const SendPage = () => {
       </Header>
       <Content>
         <SendPageAccountPicker />
-        <SendPageRecipientInput />
-        {draftTransactionExists &&
-        [SEND_STAGES.EDIT, SEND_STAGES.DRAFT].includes(sendStage) ? (
-          <SendPageContent
-            requireContractAddressAcknowledgement={
-              requireContractAddressAcknowledgement
+        {isSendFormShown && (
+          <AssetPickerAmount
+            asset={transactionAsset}
+            // TODO: update to dest asset
+            amount={amount}
+            onAssetChange={handleSelectToken}
+            onAmountChange={(newAmount) =>
+              dispatch(updateSendAmount(newAmount))
             }
           />
-        ) : (
-          <SendPageRecipient />
         )}
```

Comment:
Is the `requireContractAddressAcknowledgement` logic not needed for the src token? I think it is in the current implementation